tests/src/python/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ ADD_PYTHON_TEST(PyQgsBox3d test_qgsbox3d.py)
 ADD_PYTHON_TEST(PyQgsCategorizedSymbolRenderer test_qgscategorizedsymbolrenderer.py)
 ADD_PYTHON_TEST(PyQgsCheckableComboBox test_qgscheckablecombobox.py)
 ADD_PYTHON_TEST(PyQgsColorButton test_qgscolorbutton.py)
+ADD_PYTHON_TEST(PyQgsColorRamp test_qgscolorramp.py)
```

Comment:
There's already a test file containing lots of ramp tests as test_qgsvectorcolorramp.py -- your new tests should be merged into the existing file (and ideally renamed away from the outdated vectorcolorramp name)

Reply:
Done